bin/ovfs/src/main.rs

```diff
@@ -19,6 +19,59 @@ mod error;
 mod virtiofs;
 mod virtiofs_utils;
 
+use std::collections::HashMap;
+use std::str::FromStr;
+
+use clap::Parser;
+use opendal::{Operator, Scheme};
+use url::Url;
+
+/// User configuration for ovfs.
+#[derive(Parser, Debug)]
+#[command(version, about)]
+struct Config {
+    /// socket path used to connect with VMs
+    #[arg(env = "OVFS_MOUT_PATH", index = 1)]
+    socket_path: String,
+
+    /// location of opendal service
+    /// format: <scheme>://?<key>=<value>&<key>=<value>
+    /// example: fs://?root=/tmp
+    #[arg(env = "OVFS_BACKEND", index = 2)]
+    backend: Url,
```

Comment:
I don't like the idea of introducing a URL here. How about just using a `HashMap`? Our command line coule be like:

```shell
ovfs path/to/mount --scheme=s3 --access-key-id=xxx --secret-access-key=yyy --root=/tmp
```

Reply:
I will initiate another discussion with ofs. However, I believe ovfs is a separate project that shouldn't adhere to the same design as ofs, especially when that design seems flawed.

Reply:
I feel that we're currently pushing too aggressively to release the binary to users. Instead, we should focus on developing a library, ensuring it functions correctly and passes the tests before making it available to users.

For example, we don't need to play with the config/command line/args right now. We should make it work and than design our user interface.

Reply:
Got this. I'll draft this PR first. We should reopen this PR to discuss the user interface of the binary after the ovfs library is fully developed.